src/percy-agent-client/percy-agent-client.ts

```diff
@@ -9,28 +9,46 @@ export class PercyAgentClient {
     this.healthCheck()
   }
 
-  post(path: string, data: any) {
+  post(path: string, data: any): Promise <any> {
     if (!this.agentConnected) {
       console.warn('percy agent not started.')
-      return
+      return Promise.resolve()
```

Comment:
Would you want this to return as a failure? As-is it would return as a successfully resolved promise, but you could also return `Promise.reject()` which would signal to upstream callers that the thing was not successful.